### ISSUE TITLE
changes to about section of website

### ISSUE THREAD

**Issue by cjyetman** [christophergandrud/networkD3#173]
- [ ] 1. there's no mention of `chordNetwork()`
- [ ] 2. maybe "Radial networks with radialNetwork" should be changed to "Tree networks with diagonalNetwork and radialNetwork"

**Comment by christophergandrud** [comment]
Good idea. Though may want to hold off if you're still thinking of redoing tree?
